src/main/resources/filter/web/invisible-attributes.filter

```diff
@@ -1,6 +1,7 @@
 # Filter file for recheck with invisible and mostly irrelevant attributes 
 
 # Class can be ignored, because we track individual CSS attributes
```

Comment:
Doesn't this comment belong to `attribute=class`? Is it still needed at all?

Reply:
Wasn't that change, but Good Catch ;-)

Reply:
Yeah, I overlooked this earlier… 🙈